### DISCUSSION TITLE
[5.x] Password - Min requirements position

### DISCUSSION THREAD

**Discussion by joomleb** [joomla/joomla-cms#43382] [2. Cold Features]
Hi guys,
PHP 8.1 + Joomla 4.4.1

The **Registration page** is grabbing the "**Minimum Requirements**" from
Joomla Users Options > Password Options > Minimum Password Length... etc. and appending them below the **password1** label.

Here an image for you: https://photos.app.goo.gl/gmguWctFb2SHxBM97

The **frontend Edit Profile page** is not grabbing them for the same password1 field. 

It should, Do you agree?

**Comment by brianteeman** [comment]
Why would it. The only requirement of the password confirmation field is that it is the same.

**Comment by joomleb** [comment]
@brianteeman 
Because the 2 password fields are into the Edit Profile in case I want to change my password (they are not there to save the Edit Profile form).

Now, there are two password fields:
- **password1** - it always must meet the "Minimum Requirements" from
Joomla Users Options > Password Options > Minimum Password Length... etc.... Isn't it ?!? - if no, it would be a serious security issue to permit to a registered user to change a password as he wants (let me say 4 characterers instead of  min.10 for example).
- **password2** - (password confirmation) as you wrote it has to be the same as the password1 

So, If I'm the logged user and I want to change my password, I edit my profile and sure is quite I'm registering: I really don't remember the minimum requirements of that site !

For the same reason also the green bar at the bottom would be a must, as in the Registration form. As Frontend user I'm exactly in the same situation...

Am I missing anything?
Do you agree?

**Comment by brianteeman** [comment]
sorry I think I misunderstood the question

**Comment by joomleb** [comment]
@brianteeman
(it can happen and it's never a problem, that's why we kindly explain it to each other).

About the issue, Do you agree ?

**Comment by brianteeman** [comment]
From what I can tell this has already been fixed in j5. If you could check and confirm

**Comment by joomleb** [comment]
> From what I can tell this has already been fixed in j5. If you could check and confirm

@brianteeman you are right, tested on new J 5.0.1 installation and I can confirm it (sorry)

**Comment by joomleb** [comment]
@brianteeman 
Sorry, I'm reopening this to ask:
Cassiopeia is adding the "Min. Req." just below the password1 field label (the old password1 description position).
Would not be better to add it just below the 'green bar check" ? Like:
https://photos.app.goo.gl/yE3heDWqrEx5M6XR6
So the green bar and min. req. will be near...
Practically the "Min. Req." is quite the "green bar check description"...

**Comment by rdeutz** [comment]
Description is not a detailed that I could start developing, so converting it to discussion to define it more precise
